alea/include/alps/alea/internal/format.hpp

```diff
@@ -8,6 +8,7 @@
 #include <cassert>
 #include <ostream>
 #include <memory>
+#include <mutex>
```

Comment:
I guess you wanted to protect that call to `std::ios_base::xalloc()` but then changed your mind...

Reply:
Guilty as charged 😏 . I got confused about `mutex`es and then thought I better leave it before I make matters worse -- let's maybe discuss tomorrow.